src/p5.dance.js

```diff
@@ -104,6 +104,8 @@ module.exports = class DanceParty {
       p5Inst.preload = () => this.preload();
       p5Inst.setup = () => this.setup();
       p5Inst.draw = () => this.draw();
+      //Allows the sprite width and height to be set independently
+      this.p5_._fixedSpriteAnimationFrameSizes = true;
```

Comment:
Is this documented somewhere? Having a name starting with `_` usually means it should be treated as private, and we shouldnt really be going in and mucking with it. It would be better if p5 had some sort of API we could go through to target this.

Reply:
No documentation on this, but it's a practice I borrowed from GameLabP5: https://github.com/code-dot-org/code-dot-org/blob/30a0a9814f123b72a16661ecc9d852640dc31f1c/apps/src/gamelab/GameLabP5.js#L193

Might be worth adding a function to the p5 API, but I think that can come in a follow-up PR.